### ISSUE TITLE
[Bug]: Test run fails

### ISSUE THREAD

**Issue by CFGrote** [epi2me-labs/wf-clone-validation#15]
### What happened?

I'm trying to run the test case and get errors (see below). Does that point to a problem on my side or a bug in the code?
Any hint how to proceed would be appreciated.

My OS is debian 11 (not ubuntu 18.04 as per  dropdown below).

Thanks!

```
nextflow run epi2me-labs/wf-clone-validation --fastq test_data/test --db_directory db --no_reconcile --threads 4
```
database is located in $PWD/db, test data is in $PWD/test_data .

Complete run log below.

### Operating System

ubuntu 18.04

### Workflow Execution

Command line

### Workflow Execution - CLI Execution Profile

None

### Workflow Version

22.10.4

### Relevant log output

```shell
▶ nextflow run epi2me-labs/wf-clone-validation --fastq test_data/test --db_directory db --no_reconcile --threads 4
N E X T F L O W  ~  version 22.10.4
Launching `https://github.com/epi2me-labs/wf-clone-validation` [prickly_boyd] DSL2 - revision: 5373efbe4c [master]

WARN: Found unexpected parameters:
* --no_reconcile: true
- Ignore this warning: params.schema_ignore_params = "no_reconcile" 


||||||||||   _____ ____ ___ ____  __  __ _____      _       _
||||||||||  | ____|  _ \_ _|___ \|  \/  | ____|    | | __ _| |__  ___
|||||       |  _| | |_) | |  __) | |\/| |  _| _____| |/ _` | '_ \/ __|
|||||       | |___|  __/| | / __/| |  | | |__|_____| | (_| | |_) \__ \
||||||||||  |_____|_|  |___|_____|_|  |_|_____|    |_|\__,_|_.__/|___/
||||||||||  wf-clone-validation v0.2.7-g5373efb
--------------------------------------------------------------------------------
Core Nextflow options
  revision       : master
  runName        : prickly_boyd
  containerEngine: docker
  launchDir      : /home/XXX/Projects/Dave/wf-clone-validation
  workDir        : /home/XXX/Projects/Dave/wf-clone-validation/work
  projectDir     : /home/XXX/.nextflow/assets/epi2me-labs/wf-clone-validation
  userName       : XXX
  profile        : standard
  configFiles    : /home/XXX/.nextflow/assets/epi2me-labs/wf-clone-validation/nextflow.config

Input Options
  fastq          : test_data/test
  primers        : /home/XXX/.nextflow/assets/epi2me-labs/wf-clone-validation/data/primers.tsv

Reference Genome Options
  db_directory   : db

Sample Options
  min_barcode    : 0
  max_barcode    : 192

!! Only displaying parameters that differ from the pipeline defaults !!
--------------------------------------------------------------------------------
If you use epi2me-labs/wf-clone-validation for your analysis please cite:

* The nf-core framework
  https://doi.org/10.1038/s41587-020-0439-x


--------------------------------------------------------------------------------
This is epi2me-labs/wf-clone-validation v0.2.7-g5373efb.
--------------------------------------------------------------------------------
Checking fastq input.
Barcoded directories detected.
WARN: Excluding directories not containing .fastq(.gz) files:
WARN:    - /home/XXX/Projects/Dave/wf-clone-validation/test_data/test/barcode04
executor >  local (12)
[44/0264e6] process > pipeline:combineFastq (3)         [100%] 3 of 3 ✔
[bc/a8bc42] process > pipeline:assembleCore (1)         [100%] 1 of 1 ✔
[9b/bf0dcb] process > pipeline:medakaPolishAssembly (1) [100%] 1 of 1 ✔
[b6/971992] process > pipeline:downsampledStats (1)     [100%] 1 of 1 ✔
[bb/7d02bd] process > pipeline:findPrimers (1)          [100%] 1 of 1 ✔
[62/5ca7a7] process > pipeline:getVersions              [100%] 1 of 1 ✔
[6f/ead67c] process > pipeline:getParams                [100%] 1 of 1 ✔
[22/50d154] process > pipeline:runPlannotate (1)        [  0%] 0 of 1
[b9/899262] process > pipeline:inserts                  [100%] 1 of 1 ✔
[-        ] process > pipeline:report                   -
[89/d8ac6d] process > output (1)                        [100%] 1 of 1
Error executing process > 'pipeline:runPlannotate (1)'

Caused by:
  Process `pipeline:runPlannotate (1)` terminated with an error exit status (1)

Command executed:

  if [ -e "assemblies/OPTIONAL_FILE" ]; then
      assemblies=""
  else
      assemblies="--sequences assemblies/"
  fi
  run_plannotate.py $assemblies --database db

Command exit status:
  1

Command output:
  (empty)

Command error:
  /home/epi2melabs/conda/lib/python3.8/site-packages/plannotate/annotate.py:218: DeprecationWarning: invalid escape sequence \|
    problem_name = "pdb\|(.*)\|"
  /home/epi2melabs/conda/lib/python3.8/site-packages/validators/url.py:74: DeprecationWarning: invalid escape sequence \.
    u"(?:\.(?:[a-z\u00a1-\uffff0-9]-?)*[a-z\u00a1-\uffff0-9]+)*"
  /home/epi2melabs/conda/lib/python3.8/site-packages/validators/url.py:76: DeprecationWarning: invalid escape sequence \.
executor >  local (12)
[44/0264e6] process > pipeline:combineFastq (3)         [100%] 3 of 3 ✔
[bc/a8bc42] process > pipeline:assembleCore (1)         [100%] 1 of 1 ✔
[9b/bf0dcb] process > pipeline:medakaPolishAssembly (1) [100%] 1 of 1 ✔
[b6/971992] process > pipeline:downsampledStats (1)     [100%] 1 of 1 ✔
[bb/7d02bd] process > pipeline:findPrimers (1)          [100%] 1 of 1 ✔
[62/5ca7a7] process > pipeline:getVersions              [100%] 1 of 1 ✔
[6f/ead67c] process > pipeline:getParams                [100%] 1 of 1 ✔
[22/50d154] process > pipeline:runPlannotate (1)        [100%] 1 of 1, failed: 1 ✘
[b9/899262] process > pipeline:inserts                  [100%] 1 of 1 ✔
[-        ] process > pipeline:report                   -
[89/d8ac6d] process > output (1)                        [ 50%] 1 of 2
Error executing process > 'pipeline:runPlannotate (1)'

Caused by:
  Process `pipeline:runPlannotate (1)` terminated with an error exit status (1)

Command executed:

  if [ -e "assemblies/OPTIONAL_FILE" ]; then
      assemblies=""
  else
      assemblies="--sequences assemblies/"
  fi
  run_plannotate.py $assemblies --database db

Command exit status:
  1

Command output:
  (empty)

Command error:
  /home/epi2melabs/conda/lib/python3.8/site-packages/plannotate/annotate.py:218: DeprecationWarning: invalid escape sequence \|
    problem_name = "pdb\|(.*)\|"
  /home/epi2melabs/conda/lib/python3.8/site-packages/validators/url.py:74: DeprecationWarning: invalid escape sequence \.
    u"(?:\.(?:[a-z\u00a1-\uffff0-9]-?)*[a-z\u00a1-\uffff0-9]+)*"
  /home/epi2melabs/conda/lib/python3.8/site-packages/validators/url.py:76: DeprecationWarning: invalid escape sequence \.
    u"(?:\.(?:[a-z\u00a1-\uffff]{2,}))"
  2022-12-14 10:36:37.559 
    Warning: to view this Streamlit app on a browser, run it with the following
    command:
  
      streamlit run /home/XXX/.nextflow/assets/epi2me-labs/wf-clone-validation/bin/run_plannotate.py [ARGUMENTS]
  Traceback (most recent call last):
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/streamlit/legacy_caching/caching.py", line 540, in get_or_create_cached_value
      return_value = _read_from_cache(
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/streamlit/legacy_caching/caching.py", line 339, in _read_from_cache
      raise e
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/streamlit/legacy_caching/caching.py", line 324, in _read_from_cache
      return _read_from_mem_cache(
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/streamlit/legacy_caching/caching.py", line 242, in _read_from_mem_cache
      raise CacheKeyNotFoundError("Key not found in mem cache")
  streamlit.legacy_caching.caching.CacheKeyNotFoundError: Key not found in mem cache
  
  During handling of the above exception, another exception occurred:
  
  Traceback (most recent call last):
    File "/home/XXX/.nextflow/assets/epi2me-labs/wf-clone-validation/bin/run_plannotate.py", line 254, in <module>
      main()
    File "/home/XXX/.nextflow/assets/epi2me-labs/wf-clone-validation/bin/run_plannotate.py", line 225, in main
      tup_dic, report, plannotate_dic = attempt_annotation(
    File "/home/XXX/.nextflow/assets/epi2me-labs/wf-clone-validation/bin/run_plannotate.py", line 182, in attempt_annotation
      tup_dic, clean_df = per_assembly(sample_file, name)
    File "/home/XXX/.nextflow/assets/epi2me-labs/wf-clone-validation/bin/run_plannotate.py", line 82, in per_assembly
      plot, annotations, clean_df = run_plannotate(sample_file)
    File "/home/XXX/.nextflow/assets/epi2me-labs/wf-clone-validation/bin/run_plannotate.py", line 20, in run_plannotate
      df = annotate(
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/plannotate/annotate.py", line 344, in annotate
      blastDf = get_raw_hits(query, linear, yaml_file)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/streamlit/legacy_caching/caching.py", line 576, in wrapped_func
      return get_or_create_cached_value()
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/streamlit/legacy_caching/caching.py", line 556, in get_or_create_cached_value
      return_value = func(*args, **kwargs)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/plannotate/annotate.py", line 282, in get_raw_hits
      hits = BLAST(seq = query, db = database)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/plannotate/annotate.py", line 41, in BLAST
      inDf = parse_infernal(tmp.name)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/plannotate/infernal.py", line 12, in parse_infernal
      col_widths = [len(ele)+1 for ele in lines[1].split()]
  IndexError: list index out of range

Work dir:
  /home/XXX/Projects/Dave/wf-clone-validation/work/22/50d1545dee81f1de04bc7ed06f6ec6

Tip: view the complete command output by changing to the process work dir and entering the command `cat .command.out`
```
